spec/outputs/logentries_spec.rb

```diff
@@ -0,0 +1,2 @@
+require "logstash/devutils/rspec/spec_helper"
```

Comment:
I've check this with JRuby and MRI, and I can not reproduce this error of yours. I would recommend you to remove your Gemfile.lock and your .bundle directory and try again.